### DISCUSSION TITLE
What is the optimal font size?

### DISCUSSION THREAD

**Discussion by rarodja** [tonsky/FiraCode#1313] [Q&A] [answered]
That same, what is the recommended optimal size for the use of this font in the different environments and IDE's.

**Comment by tonsky** [comment, marked as answer]
13 pt should fit into pixel grid

**Reply by rarodja** [reply]
Thank you very much for your quick response, how much would it be in pixels?

**Reply by tonsky** [reply]
That’s an interesting (and complex!) question. This is a good summary https://tonsky.me/blog/font-size/

**Reply by mcourteaux** [reply]
I read the article. If 13pts is "fitting the pixel grid", but the conversion of pts to px is different on Windows as it is on macOS, then how can you make that statement. After reading the article, I expected the answer to be along the lines of: "13pts on macOS and 9.75pts on Windows"?
